### PR TITLE
multiple changes to deal with fiducial names

### PR DESCRIPTION
**Pull request by kaitj**
- added casefolding for case insensitive comparisons of fiducial descriptions
- replaced hard code of afids-examples submodule with variable
  - easier for local development and testing

### CODE REVIEW

**Comment by kaitj** [comment]
Addresses #25 

Quick case-insensitive comparison via `casefold`
 - Captilization not an issue during current stage
 - Fixes 3 out of 4 fiducials with capitalisation issues 
   - "splenium" has been changed to "splenium of cc" in sub module

File: fidvalidator/controller.py [hunk 1]
```diff
@@ -197,7 +198,7 @@ def index2():
         else:
             result = None
 
-    dir_contents = os.listdir('../afids-examples/')
+    dir_contents = os.listdir('AFIDS_DIR')
```
Comment (tkkuehn) [inline fidvalidator/controller.py]: Should remove the quotes here (`AFIDS_DIR`, not `'AFIDS_DIR'`).

Also, is it possible that the trailing slash being removed is going to make a difference here?

Comment (kaitj) [reply fidvalidator/controller.py]: Not as familiar with submodules, maybe @ostanley can comment.

I don't think removing the trailing slash will make a difference. And yes the quotes should not be there. Will update and push.

File: fidvalidator/model_auto.py [hunk 2]
```diff
@@ -128,11 +128,9 @@ def csv_to_json(in_csv):
 
         row_desc = parse_fcsv_field(row, 'desc', row_label)
 
-        if EXPECTED_MAP[row_label] != row_desc:
-            raise InvalidFcsvError('Row label {row_label} does not '
-                    .format(row_label=row_label)
-                     + 'match row description {row_desc}'
-                     .format(row_desc=row_desc))
+        if EXPECTED_MAP[row_label].casefold() != row_desc.casefold():
+            raise InvalidFcsvError('Row label {row_label} does not ' +
```
Comment (tkkuehn) [inline fidvalidator/model_auto.py]: I think the `InvalidFcsvError` here should have the original argument, so that the string actually gets formatted.

Comment (kaitj) [reply fidvalidator/model_auto.py]: Take a look now. I must have missed it when I merged the two. Should be fixed now.

**Pull request merged**